miniconda/Dockerfile

```diff
@@ -1,10 +1,12 @@
-FROM debian:7.4
+FROM ubuntu:14.04
```

Comment:
The ubuntu image was much larger than the simple debian image so I swapped out here.  We shouldn't be providing people with OS level support at this.  Instead, our focus should be getting conda / anaconda installed for folks to use in the smallest way possible by default.

If there's a need, I'm all for offering an ubuntu and centos flavored build, but the default should be OS agnostic and use the smallest operating system it can install on.  I would run this on busybox, but I didn't have the time to debug the install on to that system.